packages/grafana-data/src/dataframe/frameComparisons.ts

```diff
@@ -60,7 +60,9 @@ export function compareDataFrameStructures(a: DataFrame, b: DataFrame, skipConfi
         return false;
       }
 
-      if (key === 'custom') {
+      if (key === 'interval') {
+        continue;
+      } else if (key === 'custom') {
```

Comment:
```suggestion
      }
      if (key === 'custom') {
```

6 vs 1/2 dozen?  but I like to avoid else if we have a return/continue

Reply:
:exploding_head: